monitoring/snippets/metric_create_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCreateCustomMetric(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	buf := &bytes.Buffer{}
+	metric, err := createCustomMetric(buf, tc.ProjectID, metricType)
+	if err != nil {
+		t.Fatalf("createCustomMetric: %v", err)
+	}
+	defer deleteMetric(ioutil.Discard, metric.GetName())
+
+	want := "Created"
+	if got := buf.String(); !strings.Contains(got, want) {
```

Comment:
naive question: why strings.Contains instead of equality?

Reply:
This is mostly to check the snippet finished and printed some sort of success message. The exact content doesn't matter as much.